ui/overview_page.go

```diff
@@ -366,28 +458,13 @@ func (pg *overviewPage) syncStatusColumn(gtx layout.Context) layout.Dimensions {
 // drawlayout wraps the page tx and sync section in a card layout
 func (pg *overviewPage) drawlayout(gtx layout.Context, body layout.Widget) layout.Dimensions {
 	return pg.theme.Card().Layout(gtx, func(gtx C) D {
-		return layout.UniformInset(values.MarginPadding20).Layout(gtx, body)
+		return layout.UniformInset(values.MarginPadding16).Layout(gtx, body)
 	})
 }
 
-// syncingContent lays out sync status content when the wallet is syncing
-func (pg *overviewPage) syncActiveContent(gtx layout.Context, uniform layout.Inset) layout.Dimensions {
-	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-		layout.Rigid(func(gtx C) D {
-			return pg.progressBarRow(gtx, uniform)
-		}),
-		layout.Rigid(func(gtx C) D {
-			return pg.progressStatusRow(gtx, uniform)
-		}),
-		layout.Rigid(func(gtx C) D {
-			return pg.walletSyncRow(gtx, uniform)
-		}),
-	)
-}
-
 // syncDormantContent lays out sync status content when the wallet is synced or not connected
-func (pg *overviewPage) syncDormantContent(gtx layout.Context) layout.Dimensions {
-	return layout.Inset{Left: values.MarginPadding45}.Layout(gtx, func(gtx C) D {
+func (pg *overviewPage) syncDormantContent(gtx layout.Context, uniform layout.Inset) layout.Dimensions {
+	return uniform.Layout(gtx, func(gtx C) D {
 		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 			layout.Rigid(func(gtx C) D {
 				return layout.Inset{Bottom: values.TextSize12}.Layout(gtx, func(gtx C) D {
```

Comment:
TextSize dimension shouldn't be used for margin or padding. Kindly define an equivalent of this dimension as MarginPadding